hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py

```diff
@@ -63,27 +63,23 @@ def _create_hr_timesheet(self):
         )
 
     def test_hr_timesheet_without_tags(self):
-        self.task.analytic_account_id = self.analytic_account_1
```

Comment:
Why removing this?

Reply:
Yeah, but then, `analytic_account_1` is never used. If not needed, remove its creation.

Reply:
Done! Thanks!